src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/SingleMethodCompilationModuleGroup.cs

```diff
@@ -29,6 +31,11 @@ public override bool ContainsType(TypeDesc type)
             return type == _method.OwningType;
         }
 
+        public override bool VersionsWithModule(ModuleDesc module)
+        {
+            return ((EcmaMethod)_method).Module == module;
```

Comment:
Should we use GetTypicalMethodDefinition?

Reply:
I think _method and its typical definition will have the same module

Reply:
It will but a generic method won't cast to an EcmaMethod

Reply:
Ah, good catch. I'll fix that.